MREUnityRuntime/MREUnityRuntimeLib/API/MREApi.cs

```diff
@@ -139,10 +139,11 @@ public class MREAppsAPI
 		/// </summary>
 		/// <param name="ownerScript">The owner unity script for the app.</param>
 		/// <param name="globalAppId">The global app id for the app being instanced, or empty if there is no global app id.</param>
+		/// <param name="ephemeralAppId">A string uniquely identifying the MRE instance on all clients.</param>
 		/// <returns>Returns the newly created mixed reality extension app.</returns>
```

Comment:
While this is going to make the description longer, would like to change to clarify that this value should be identical across all clients talking together, but should ideally change at regular intervals (for example, if host app has a multiplayer server instance that synchronize users, then instance unique ID)?) Clarify that it's important it changes to support users that decline user tracking)?

Maybe for globalAppID, add in that the global app id should be persistent

Reply:
Agree that the documentation header should include the necessity of the ID being rotated or ephemeral in the use case of a user denying tracking.